src/components/Mote.tsx

```diff
@@ -15,7 +15,7 @@ const Mote = ({ mote }: MoteProps): ReactElement => {
   const virksomhetQuery = useVirksomhetQuery(virksomhetsnummer);
 
   const virksomhetsNavn = (): string => {
-    if (virksomhetQuery.isLoading) {
+    if (virksomhetQuery.isInitialLoading) {
```

Comment:
Endret til isInitialLoading for queries som kan være disabled (Ref: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#disabled-queries https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading)